src/main/java/com/teamscale/upload/utils/FileSystemUtils.java

```diff
@@ -135,6 +137,25 @@ private static void ensureEmptyDirectory(File directory) throws IOException {
 		}
 	}
 
+	/**
+	 * Ensures that the file represents an empty directory. Creates the directory if
```

Comment:
`Ensures that the file represents an empty directory.` I think that is not what this method does ;)
It ensures that the given file is below (in or in a sub-directory) of the given directory, right?

Do we even need to ensure here that either of them actually exists?

Reply:
Ahh sorry 🤦🏼‍♂️ I changed this comment at some point, but had to fork the repository before creating the pull request and something most gone wrong while copying
I adjusted it, thanks!